miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -2,29 +2,62 @@ package com.rakuten.tech.mobile.miniapp.js
 
 import android.webkit.JavascriptInterface
 import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
 import com.rakuten.tech.mobile.miniapp.display.WebViewListener
 
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
 /** Bridge interface for communicating with mini app. **/
 abstract class MiniAppMessageBridge {
     private lateinit var webViewListener: WebViewListener
 
     /** Get provided id of mini app for any purpose. **/
     abstract fun getUniqueId(): String
 
+    /** Post permission request from external. **/
+    abstract fun requestPermission(
+        callbackId: String,
+        miniAppPermissionType: String,
```

Comment:
I think we should pass a custom type rather than a string. Like maybe a `MiniAppPermissionType` enum?